net/sing-box/Makefile

```diff
@@ -96,6 +107,9 @@ PKG_CONFIG_DEPENDS:= \
 	CONFIG_SINGBOX_WITH_V2RAY_API \
 	CONFIG_SINGBOX_WITH_WIREGUARD
 
+ifeq ($(BUILD_VARIANT),tiny)
+GO_PKG_TAGS:=with_gvisor,with_quic,with_dhcp,with_wireguard,with_utls,with_acme,with_clash_api
```

Comment:
I would suggest using fixed tags for full version (same as upstream defaults), and instead make tiny version customizable.